docker-compose.geoserver.yml

```diff
@@ -0,0 +1,37 @@
+version: "3.2"
+
+services:
+
+  geoserver:
+    image: clowder/geoserver:${VERSION:-latest}
+    networks:
+      - clowder
+#    user: "55242:17027"
+    environment:
+      - RABBITMQ_URI=${RABBITMQ_URI:-amqp://guest:guest@rabbitmq/%2F}
+      - RABBITMQ_EXCHANGE=${RABBITMQ_EXCHANGE:-clowder}
+      - REGISTRATION_ENDPOINTS=${REGISTRATION_ENDPOINTS}
+      - PROXY_ON=${PROXY_ON}
+      - PROXY_HOST=${PROXY_HOST}
+      - PROXY_URL=${PROXY_URL}
+      - GEOSERVER_USERNAME=${GEOSERVER_USERNAME:-admin}
```

Comment:
can you add these to the env.example and document them

Reply:
Yes, can do.